tests/modules/services/dropbox/basic-configuration.nix

```diff
@@ -0,0 +1,10 @@
+{ config, pkgs, ... }:
+
+{
+  config = {
+    services.dropbox = {
+      enable = true;
+      path = "${config.home.homeDirectory}/dropbox";
+    };
+  };
```

Comment:
You'll need to add a test script. Also add an overlay to mock the `dropbox-cli` package. Search for `outPath` in the test directory for examples.

Reply:
Done, but test do not pass. For some reason. it does not activate the service on my computer, but I can see it in the build output? Can you look at the latest and maybe see why?

I have no clue atm.

Reply:
The tests should not activate the service, the Home Manager test suite only supports golden tests so you can, for example, verify that the generated service file is as expected.

See https://github.com/rycee/home-manager/blob/master/tests/modules/services/lieer/lieer-service.nix for an example of verifying systemd units.

Reply:
But as far as I can see in the code, I am doing the samy by having

```
    nmt.script = ''
      serviceFile=home-files/.config/systemd/user/dropbox.service
      assertFileExists $serviceFile
    '';
```

Or am I wrong? Also, when using the service in general after the changes, it doesn't seem to put the service file into my `~/.config/systemd/user/` dir, which I cannot see why it should not. Sadly..

Reply:
Ah, right! You need to add
```nix
(loadModule ./services/dropbox.nix { condition = hostPlatform.isLinux; })
```
to `modules/modules.nix`.

Reply:
Perfect, thanks! Everything is fixed now it seems!